packages/yew-macro/tests/html_macro/html-component-pass.rs

```diff
@@ -5,11 +5,14 @@ use yew::html::ChildrenRenderer;
 use yew::prelude::*;
 use yew::virtual_dom::{VChild, VNode};
 
-pub struct Generic<G> {
-    marker: PhantomData<G>,
+pub struct Generic<T> {
```

Comment:
Could you move this to a new file too? A lot of tests for the macro are currently in the same file which leads to messy diffs in case someone changes them. Like all the unrelated line changes in yours for example.
By splitting these cases into separate files we can mitigate this somewhat.

I would recommend a filename like `generic-component-pass.rs` (If you really want you can add the `html-` prefix but I'd like to get rid of it for all files at some point anyway because it's redundant).

Please also add the corresponding `component-generic-fail.rs` and `component-generic-fail.stderr`.

Reply:
If it's ok, I'll just remove all the `html-` prefixes

Reply:
oh please do